frontends/concrete-python/concrete/fhe/compilation/utils.py

```diff
@@ -7,6 +7,8 @@
 import re
 from copy import deepcopy
 from typing import Any, Callable, Dict, Iterable, List, Optional, Set, Tuple, Union
+import traceback, os
+from concrete import fhe
```

Comment:
internal import are usually (in CP) after standard libs import and external libs import

Reply:
Okay changed.